jobs/telemetry_batch_view.py

```diff
@@ -22,6 +22,42 @@ def retrieve_jar():
     if jar_url is None:
         exit(1)
 
+
+    print("Retrieving JAR: {}".format(jar_url))
```

Comment:
Do we have a habit of `print`ing in this project? Should we instead call `logging.info`?

Reply:
Currently, yes. I'd be fine with switching to `logging.info`, but I would prefer we tackle that in a follow-up.